examples/guides/se-restful-webservice/README.adoc

```diff
@@ -250,208 +200,180 @@ import io.helidon.webserver.ServerRequest;
 import io.helidon.webserver.ServerResponse;
 import io.helidon.webserver.Service;
 ----
-
-. Make `GreetService` implement `io.helidon.webserver.Service`.
-. Set two static fields by reading the config file and setting the initial greeting from
-the loaded config:
+for JSON and config support and for the key parts of the web server.
```

Comment:
"These imports are necessary for JSON and ..."

Reply:
Done